README.md

```diff
@@ -1,9 +1,19 @@
 <img src="https://www.kubeflow.org/images/logo.svg" width="100">
-Kubeflow is a Cloud Native platform for machine learning based on Google’s internal machine learning pipelines.
+Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
 
 ---
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
 
+The Kubeflow community is organized into working groups (WGs) with associated repositories, that focus on specific pieces of the ML platform. 
+
+## Working Groups
+* [AutoML](https://github.com/kubeflow/community/tree/master/wg-automl)
```

Comment:
This is great!
Can we also add main repos for each WG where they prefer for github issues?
and recommend people creating issues for individual repos

e.g. for Pipelines, I'd suggest putting down:
* kubeflow/pipelines
* kubeflow/kfp-tekton

/cc @animeshsingh

Reply:
What would be a good structure? 

* [Pipelines](https://github.com/kubeflow/community/tree/master/wg-pipelines) - [Code](https://github.com/kubeflow/pipelines)

Or 

* Pipelines - [Working Group](https://github.com/kubeflow/community/tree/master/wg-pipelines) - [Code](https://github.com/kubeflow/pipelines)

?

Reply:
@RFMVasconcelos 

this is the 2nd repo - https://github.com/kubeflow/kfp-tekton


Additionally @Bobgy couple of comments on this
1. Charter has disappeared from the WG folder https://github.com/kubeflow/community/tree/master/wg-pipelines
2. Would like to add my name there for KFP-Tekton project, so that folks know who to connect for that

Reply:
@RFMVasconcelos can't we just include that information in the `kubeflow/community` repo?

Keeping the current links you made, e.g:`https://github.com/kubeflow/community/tree/master/wg-notebooks`

Reply:
@thesuperzapper I agree, this way we only redirect here for simplicity.